### ISSUE TITLE
Inview not completing graphic

### ISSUE THREAD

**Issue by webb-nickj** [adaptlearning/adapt_framework#1467]
I have come across an issue where trickle and inview seem to be conspiring to prevent graphics completing.

This is happening from content produced in the authoring tool, with trickle set to act at the article level. The trickle button and auto-scrolling are enabled. Trickle is turned off at the block level. 

When the screen is wide enough so minimal scrolling occurs (~150px in this example) when the second article is revealed, the graphic component (bottom left) is not being set to complete. Manually calling $.inview() is also not triggering completion.

Adding min-height (e.g. 80vh) to the articles resolves the issue.

The graphic has the no-state class added, but removing this doesn't change the behaviour.

![image](https://cloud.githubusercontent.com/assets/19776091/23945263/cbc3ec6a-096d-11e7-9d36-6f6c89863b20.png)

![image](https://cloud.githubusercontent.com/assets/19776091/23945294/e0ab2116-096d-11e7-8cd6-62c31fdd310c.png)

**Comment by oliverfoster** [comment]
is this still an issue @webb-nickj

**Comment by webb-nickj** [comment]
Not sure to be honest Oli - we replaced trickle entirely in that project anyway.

**Comment by oliverfoster** [comment]
cool. what did you replace it with?

**Comment by webb-nickj** [comment]
@oliverfoster - we added something in as part of the theme. Because we are only controlling flow at article level we replaced it with something that applies absolute positioning to the articles to me them out of flow and offscreen when they shouldn't be seen. Then there's no need to keep changing the wrapper height.  It also means there's no risk of content being cropped by trickle (especially when using CSS transformations).
